components/announcement_bar/text_dismissable_bar.jsx

```diff
@@ -59,7 +60,7 @@ export default class TextDismissableBar extends React.PureComponent {
                 handleClose={this.handleDismiss}
                 message={
                     <span
-                        dangerouslySetInnerHTML={{__html: TextFormatting.formatText(text, {singleline: true, mentionHighlight: false})}}
+                        dangerouslySetInnerHTML={{__html: TextFormatting.formatText(text, {singleline: true, mentionHighlight: false}, this.props.emojiMap)}}
```

Comment:
Instead of passing the `emojiMap` around here, you might be able to look at using the `Markdown` component for the `message` instead of calling `formatText` directly